code/_onclick/hud/screen_objects.dm

```diff
@@ -23,6 +23,8 @@
 	master = null
 	return ..()
 
+/obj/screen/orbit(atom/A, radius, clockwise, rotation_speed, rotation_segments, pre_rotation, lockinorbit)

```

Comment:
uh, no, they meant ```/orbit()```, you don't need to pass in the args at all if you're just ignoring them.